src/Keboola/GeocodingAugmentation/Augmentation.php

```diff
@@ -209,7 +205,7 @@ protected function setupProvider($httpAdapter, $config)
         }
         switch ($config['provider']) {
             case 'google_maps':
-                return new GoogleMaps($httpAdapter, $locale, null, true, $config['apiKey']);
+                return new GoogleMaps($httpClient, null, $config['apiKey']);
```

Comment:
Vypadá to že potichu všude zmizelo využití `locale`. Z konfigurace se nově jenom přečte a pak zapíše do resultu. 
To taky vypadá na BC break. A pokud se nepoužívá tak nedává smysl ho  vyžadovat v parametrech. Proč si ho vlastně dával pryč?

Reply:
To je změna v tý Geocoder knihovně, locale se zadává až u query: https://github.com/keboola/geocoding-augmentation/blob/ad37d4f56324f04921fbd6f9f9f7a4c539d93601/src/Keboola/GeocodingAugmentation/Augmentation.php#L114 (tady v konstruktoru provideru se dřív zadával region, ale už to nepodporují. Resp. podporuje jen Google, ale zkoušel jsem že to ani nefunguje. (Nejspíš proto, že regiony mají jiný kódy než locale))

Reply:
Ale tady to co linkuješ je už jenom předávání do resultu který se uloží ne? To předání do query nějak nevidím.

Reply:
Tvl, já jsem úplně blbej. Tak to moment.

Reply:
Mnó, tak jsem tam musel přistrčit takovej interface mezi `league/geotools` a `willdurand/geocoder`. My používáme `geotools`, která rozšiřuje `geocoder` o paralelní volání (tj. to https://github.com/keboola/geocoding-augmentation/blob/3b6eb2e893036abb442107a1583e8d5e11526908/src/Keboola/GeocodingAugmentation/Augmentation.php#L85). Jenže ty provider classy u novýho `geocoderu` používají jiný interface (`\Geocoder\Provider`) než dovoluje použít `geotools` (`\Geocoder\Geocoder`). A tomu odpovídá jen ten `\Geocoder\ProviderAggregator`, proto jsme ho tam používali jako prostředníka. Jenže ten zase neumí předat locale, tak jsem ho trochu přiohnul (https://github.com/keboola/geocoding-augmentation/blob/jakub-libs-update/src/Keboola/GeocodingAugmentation/ProviderAggregator.php). Uf, doufám že je to srozumitelný. ;o)

Reply:
Jo zrovna jsem se v tom vrtal než jsi to napsal a říkal jsem si proč rovnou neimplmentuješ ten `Geocoder` a pak jsem si všimnul že ten `Batch` bohužel vyžaduje ten `ProviderAggregator`.
Takže imho ok  takhle pokud to nechceme celý nějak překopávat.